plugins/package-managers/pub/src/test/kotlin/utils/PubCacheReaderTest.kt

```diff
@@ -59,7 +59,6 @@ class PubCacheReaderTest : WordSpec({
                         {
                             "dependency": "direct main",
                             "description": {
-                                "ref": "master",
```

Comment:
Is it really much simpler to remove this compared to adding `ref` to `PackageInfo.Description` for completeness? I somehow liked to be able to get an overview of unresolved vs resolved ref even if it's unused currently.

Or actually, wouldn't the code just with without this change and without adding `ref` to `PackageInfo.Description` as you have `@JsonIgnoreProperties(ignoreUnknown = true)`?

Reply:
> Is it really much simpler to remove this compared to adding ref to PackageInfo.Description for completeness? 

It has to be removed anyway, because from the following commit onwards, the test no more operates on JSON input.

> I somehow liked to be able to get an overview of unresolved vs resolved ref even if it's unused currently.
...
> Or actually, wouldn't the code just with without this change and without adding ref to PackageInfo.Description as you have @JsonIgnoreProperties(ignoreUnknown = true)?

Yes, I've prefer to use ignore unkown and not add unused properties.

Reply:
> Yes, I've prefer to use ignore unkown and not add unused properties.

Just to share my thinking here: That makes it a bit hard to get an overview over the features / metadata a package manager provides in order to maybe implement new features in the future.

But I'm ok with keeping this as it is in the PR now.